fabric-rendering-v1/src/client/resources/fabric-rendering-v1.accesswidener

```diff
@@ -0,0 +1,6 @@
+accessWidener	v2	named
+
+accessible method net/minecraft/client/render/Camera moveBy (DDD)V
```

Comment:
Please use an accessor mixin in the testmod instead - creating a new AW adds a MC jar to the gradle project, so it should be avoided when possible. ;)

Reply:
That doesn't sound very user friendly, since it would require every user of the api to create a camera accessor to modify its transformation. Should a public facing accessor mixin be preferred over an AW?

Reply:
This access widener would not be exposed to users of this API anyway as it's not transitive.

Reply:
fair point, I forgot I moved it